Documentation/project-docs/dogfooding.md

```diff
@@ -116,16 +116,16 @@ make it self-contained
 ```XML
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeFrameworkVersion>2.1.0-preview1-25825-07</RuntimeFrameworkVersion> <!-- modify build in this line -->
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <RuntimeFrameworkVersion>3.0.0-preview-27218-01</RuntimeFrameworkVersion> <!-- modify build in this line -->
     <RuntimeIdentifier>win-x64</RuntimeIdentifier> <!-- make self-contained -->
   </PropertyGroup>
 ```
 
 ```
 $ dotnet restore
 $ dotnet publish
-$ bin\Debug\netcoreapp2.1\win-x64\publish\App.exe
+$ bin\Debug\netcoreapp3.0\win-x64\publish\App.exe
 ```
 
 ## More Advanced Scenario - Using your local CoreFx build
```

Comment:
I think `build -allconfigurations` should now be `build -buildallconfigurations` for some reason..

Reply:
you'are right
```
λ build -?
build.ps1 [-ArchGroup <string>] [-Release] [-TargetGroup <string>] [-OSGroup <string>] [-BuildAllConfigurations] [-RuntimeOS <string>] [-OuterLoop] [-SkipTests] [-Restore] [-Clean] [-BuildTests] [-Test] [-Coverage] [-InitTools] [-ExtraArgs <string[]>] [<CommonParameters>]
```
however I did `build -allconfigurations` and it generates packages...maybe not all.

Reply:
done